packages/inventory-compliance/src/index.js

```diff
@@ -1,5 +1,5 @@
 export { default as default } from './Compliance';
-export { default as SystemRulesTable, selectColumns as selectRulesTableColumns } from './SystemRulesTable';
+export { default as SystemRulesTable, selectColumns } from './Components/RulesTable';
```

Comment:
This replaces the current `SystemRulesTable`, the PR does not for now remove the actual files from the old rules table.